keanu-python/keanu/base.py

```diff
@@ -14,22 +13,25 @@ def __repr__(self) -> str:
         return "[{0} => {1}]".format(self._class, type(self))
 
     def __getattr__(self, k: str) -> Callable:
-        python_name = _to_snake_case_name(k)
+        self.__check_if_unwrapped(k)
+        self.__check_if_java_object_has_attr(k)
 
-        if k != python_name:
-            if python_name in self.__class__.__dict__:
-                raise AttributeError("{} has no attribute {}. Did you mean {}?".format(self.__class__, k, python_name))
+        return self.unwrap().__getattr__(k)
```

Comment:
Are you aware that you can call all java methods (in camel case) on the `JavaObjectWrapper` without calling `unwrap`? i.e. this test passes:

    def test_you_can_call_a_java_method_on_the_wrapped_object(java_list_wrapper) -> None:
        java_list_wrapper.add(100)
        assert java_list_wrapper.size() == 4
        assert not java_list_wrapper.isEmpty()